components/screens/contracts/ContractScreen.tsx

```diff
@@ -275,6 +305,44 @@ class ContractScreen extends React.Component<Props, State> {
     );
   }
 
+  /**
+   * Returns true if all fields of area detail values are filled
+   * @param areaDetailValues area detail values
+   */
```

Comment:
```suggestion
  /**
   * Returns true if all fields of area detail values are filled
   * @param areaDetailValues area detail values
   * @returns true if all fields are filled, otherwise false
   */
```